### ISSUE TITLE
multiple index file search option?

### ISSUE THREAD

**Issue by byhub** [OrchardCMS/Orchard#8324]
is there any api or way to handle multiple index files search at once? I mean not merging files directly but searching all of them at the same time and return subset of all results merged? I see that there is MultiReader option for Lucene but not sure if it is implemented in ISearch builder.  Do we have to change these orchard core files to get what we need?  or is there any builtin method already?
Thx in advance

**Comment by sebastienros** [comment]
Did you mean to file the issue in the Orchard Core repository?

**Comment by byhub** [comment]
No, we are using orchard 1.9.2. When I typed "orchard core", I meant to say Orchard 1x builtin API's... sorry for misunderstanding.  Since I couldnt find anything in exisitng codebase,  I have implemented new method for multireader option.
